src/lib/statistics/chunk_statistics/histograms/histogram_utils.cpp

```diff
@@ -0,0 +1,193 @@
+#include "histogram_utils.hpp"
+
+#include <cmath>
+
+#include <algorithm>
+#include <limits>
+#include <string>
+#include <utility>
+
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+std::string next_value(const std::string& value, const std::string& supported_characters,
+                       const uint32_t string_prefix_length) {
```

Comment:
Are these "utils" usable from anywhere outside the histogram? This looks like it doesn't make sense outside of the histogram that defines the supported characters and the prefix length.

Reply:
Yes, they can be used in other parts of the statistics component, at least they are in my local branches.
I'd wait for the statistics rewrite and see if we need them there. If not we can integrate them into the histograms. Does that work for you?

Reply:
If we can make sure that this does not get lost, ok.